packages/e2e/test/wallet/SingleAddressWallet/nft.test.ts

```diff
@@ -145,29 +140,9 @@ describe('SingleAddressWallet.assets/nft', () => {
         otherProperties: new Map([['id', '1']]),
         version: '1.0'
       },
-      otherProperties: undefined,
       policyId: 'e80c05f27dec74e8c04f27bdf711dff8ae03167dda9b7760b7d92cef',
       quantity: 1n,
-      tokenMetadata: {
-        desc: 'NFT with different types of files',
-        description: 'NFT with different types of files',
-        files: [
-          {
-            mediaType: 'video/mp4',
-            name: 'some name',
-            src: 'file://some_video_file'
-          },
-          {
-            mediaType: 'audio/mpeg',
-            name: 'some name',
-            src: ['file://some_audio_file', 'file://another_audio_file']
-          }
-        ],
-        icon: 'ipfs://somehash',
-        id: '1',
-        mediaType: 'image/png',
-        name: 'NFT with files'
-      }
+      tokenMetadata: null
```

Comment:
I remember this assertion use to work, what changed that made the tokenMetadata now null?

Reply:
It was a bug that was fixed in https://github.com/input-output-hk/cardano-js-sdk/pull/345, but this e2e test assertion wasn't updated.
![image](https://user-images.githubusercontent.com/303881/183566356-3c4dcc9e-7e9d-44c2-8f06-985733430484.png)

I've checked there is no result when manually querying the off-chain token metadata server, so `null` is the expected result.